### PR TITLE
fix(frame_index): making rerun's "frame_index" timeline compatible with behaviour1k datasets

### PR DESCRIPTION
**Pull request by CarolinePascal**
Supersedes #2872 - Thank you @fwd4 for the fix !

## Type / Scope

- **Type**: Bug
- **Scope**: lerobot-dataset-viz

## Summary / Motivation

- Because behavior1k datasets do not have a `frame_index` feature, `lerobot-dataset_viz` crashes.

## Related issues

- Fixes / Closes: # (if any)
- Related: # (if any)

## What changed

- The `frame_index` pipeline is now defined according to `index` which is present in all datasets. The episode-wise normalization (first frame is at index 0) is achieved with the first index of the episode.

## How was this tested (or how to run locally)

No regression :
  ```bash
lerobot-dataset-viz --repo-id "lerobot/pusht"  --episode-index 10              
 ```

Fixed feature :
  ```bash
lerobot-dataset-viz --repo-id "erobot/behavior1k-task0000"  --episode-index 10              
 ```

## Checklist (required before merge)

- [x] Linting/formatting run (`pre-commit run -a`)
- [x] All tests pass locally (`pytest`)
- [x] Documentation updated
- [x] CI is green

## Reviewer notes

- Anything the reviewer should focus on (performance, edge-cases, specific files) or general notes.
- Anyone in the community is free to review the PR.

### CODE REVIEW

File: src/lerobot/scripts/lerobot_dataset_viz.py [hunk 1]
```diff
@@ -132,10 +132,11 @@ def visualize_dataset(
 
     logging.info("Logging to Rerun")
 
+    first_index = dataset[0]["index"].item()
     for batch in tqdm.tqdm(dataloader, total=len(dataloader)):
         # iterate over the batch
         for i in range(len(batch["index"])):
-            rr.set_time("frame_index", sequence=batch["frame_index"][i].item())
+            rr.set_time("frame_index", sequence=batch["index"][i].item() - first_index)
```
[the remark this answers is not in the document]
Comment (CarolinePascal) [reply src/lerobot/scripts/lerobot_dataset_viz.py]: Resolved in [a552345](https://github.com/huggingface/lerobot/pull/3068/commits/a5523459c74198860bdbf01c99ec7327e2844e40)

**Review by pkooij — APPROVED** [review]
